docker/Makefile

```diff
@@ -1,10 +1,17 @@
+FALCON_VERSION=$(shell sudo docker run -it falconry/falcon-bench:py3 python3 -c 'import falcon; import sys; sys.stdout.write(falcon.__version__)')
+
 all: build-benchmark-images
 
 build-benchmark-images:
-	sudo docker pull python:3.7-slim
-	sudo docker pull python:3.7
+	sudo docker pull python:3.8-slim
+	sudo docker pull python:3.8
```

Comment:
why not 3.9?

Reply:
@kgriffs confirm please, but my personal take on this is that our development of 3.0, as well as the majority of the CI gates, are based upon 3.8, so it makes sense to present what has been our primary platform, as opposed to CPython 3.9 that was released late in the 3.0 cycle (already after alphas).